source/session.ts

```diff
@@ -688,6 +752,7 @@ export class Session {
    * @return {Object|null} Schema definition
    */
   getSchema(schemaId: string): Schema | null {
```

Comment:
If we do, would it be as a private attribute? I don't think I want to expand the scope of the api client and make it diverge from the python api more than necessary?

Reply:
The map itself, sure. Could you still replace all it's usage in data-utils by this getSchema method?

Reply:
i have now implemented the map. since it's not public, we have to still do the same mapping in the frontend codebase, so no change is necessary. please review.